6238_prototype/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -69,12 +69,11 @@ private void addBall() {
     ballSubsystem.setDefaultCommand(ballManualCommand);
```

Comment:
This should not compile since you deleted the `BallManualCommand` class.  If you have attempted to build the code, you should do that and fix any errors.

We don't need a default command for the ball subsystem any longer so the call to `setDefaultCommand` can go.